<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowRegistrator</class>
 <widget class="QMainWindow" name="WindowRegistrator">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>894</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>894</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Регистратор</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/newPrefix/Registro.ico</normaloff>:/newPrefix/Registro.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>304</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="cbTime">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>19</y>
      <width>74</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>635</x>
      <y>22</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Коэффициент развёртки</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbU0">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>51</y>
      <width>74</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>5mV/дел</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10mV/дел</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20mV/дел</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50mV/дел</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100mV/дел</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200mV/дел</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>500mV/дел</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1V/дел</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2V/дел</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5V/дел</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>635</x>
      <y>54</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Диап кан 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>635</x>
      <y>86</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Диап кан 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>636</x>
      <y>225</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Порт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbStart">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>120</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Старт</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="cbPort">
    <property name="geometry">
     <rect>
      <x>679</x>
      <y>222</y>
      <width>74</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbUpdatePorts">
    <property name="geometry">
     <rect>
      <x>774</x>
      <y>220</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Обновить</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>601</width>
      <height>304</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="cbU1">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>82</y>
      <width>74</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>5mV/дел</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10mV/дел</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20mV/дел</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50mV/дел</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100mV/дел</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200mV/дел</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>500mV/дел</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1V/дел</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2V/дел</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5V/дел</string>
     </property>
    </item>
   </widget>
   <widget class="QScrollBar" name="hsbCh0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>313</y>
      <width>601</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="pageStep">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="hsbCh1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>653</y>
      <width>601</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="pageStep">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pbDir">
    <property name="geometry">
     <rect>
      <x>716</x>
      <y>120</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Каталог</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbOpen">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>120</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="cbLengthRecord">
    <property name="geometry">
     <rect>
      <x>722</x>
      <y>158</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>629</x>
      <y>160</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Длина записи</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>270</y>
      <width>251</width>
      <height>95</height>
     </rect>
    </property>
    <property name="title">
     <string>Канал 1</string>
    </property>
    <widget class="QCheckBox" name="chbUCursCh0">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>24</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>21</y>
       <width>92</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>U1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_5">
        <property name="horizontalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLineEdit" name="leUCur0Ch0">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="sbUCur0Ch0">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>18</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>18</width>
            <height>22</height>
           </size>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>0</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>146</x>
       <y>21</y>
       <width>92</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>U2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_8">
        <property name="horizontalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLineEdit" name="leUCur1Ch0">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="sbUCur1Ch0">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>18</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>18</width>
            <height>22</height>
           </size>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>0</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>146</x>
       <y>55</y>
       <width>73</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>dU</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_6">
        <property name="horizontalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLineEdit" name="leDeltaUCurCh0">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>370</y>
      <width>251</width>
      <height>95</height>
     </rect>
    </property>
    <property name="title">
     <string>Канал 2</string>
    </property>
    <widget class="QCheckBox" name="chbUCursCh1">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>24</y>
       <width>16</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>21</y>
       <width>92</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>U1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_10">
        <property name="horizontalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLineEdit" name="leUCur0Ch1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="sbUCur0Ch1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>18</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>18</width>
            <height>22</height>
           </size>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>0</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>146</x>
       <y>21</y>
       <width>92</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>U2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_9">
        <property name="horizontalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLineEdit" name="leUCur1Ch1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="sbUCur1Ch1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>18</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>18</width>
            <height>22</height>
           </size>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>0</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>146</x>
       <y>55</y>
       <width>73</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>dU</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_7">
        <property name="horizontalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLineEdit" name="leDeltaUCurCh1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>894</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="aExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Настройки</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>Цвет</string>
     </property>
     <addaction name="aColorBackground"/>
     <addaction name="aColorGrid"/>
     <addaction name="aColorSignal"/>
    </widget>
    <addaction name="menu_3"/>
    <addaction name="aCalibrate"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="aExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="aColorBackground">
   <property name="text">
    <string>Фон</string>
   </property>
  </action>
  <action name="aColorGrid">
   <property name="text">
    <string>Сетка</string>
   </property>
  </action>
  <action name="aColorSignal">
   <property name="text">
    <string>Сигнал</string>
   </property>
  </action>
  <action name="aCalibrate">
   <property name="text">
    <string>Калибровать</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
